src/Routes/CommentRoute.js

```diff
@@ -0,0 +1,12 @@
+import express from "express";
+import {list, show, create} from "../controllers/CommentController";
+
+const router = express.Router();
+
+router.get("/comments", list);
+
+router.get("/comments:id", show);
```

Comment:
Your routes for get by id are missing one single character in the path. What is it?